overlord/ifacestate/helpers.go

```diff
@@ -98,6 +101,28 @@ func (m *InterfaceManager) addSnaps() error {
 	return nil
 }
 
+// fixDisconnectedCorePlugs will re-connect the network-bind-plug and
+// core-support-plug to the corresponding slots on the core snap.
+// This cures the effects of bug https://bugs.launchpad.net/snappy/+bug/1680097
+func (m *InterfaceManager) fixDisconnectedCorePlugs() error {
```

Comment:
do I understand correctly that either this will do something or #3154,  the latter will do something only if there are connections but with the old names?